module/modmanager/manager.go

```diff
@@ -1121,7 +1121,7 @@ func (m *module) startProcess(
 	moduleWorkingDirectory, ok := moduleEnvironment["VIAM_MODULE_ROOT"]
 	if !ok {
 		moduleWorkingDirectory = filepath.Dir(absoluteExePath)
-		logger.CWarnw(ctx, "VIAM_MODULE_ROOT was not passed to module. Defaulting to module's working directory",
```

Comment:
Not really a warning and not particularly interesting to most users IMO.

Reply:
Agreed this is not a useful log line. The information that viam module root was not specified can be useful. But I'd rather have a startup log line that just tacks on all of the viam related env variables and their values.

No need to do in this PR.

Reply:
Agreed! https://viam.atlassian.net/browse/RSDK-9326.